test/592-add-adjust-bicycle-pois.py

```diff
@@ -22,8 +22,8 @@
 assert_has_feature(
     16, 17238, 25950, 'pois',
     { 'kind': 'bicycle_rental_station', 'min_zoom': 17,
-      'capacity': 19, 'network': 'Bike Chattanooga',
-      'operator': 'Alta Bike Share', 'ref': '5' })
+      'capacity': 15, 'network': 'Bike Chattanooga',
+      'operator': 'Alta Bike Share', 'ref': '24' })
 
```

Comment:
Nice catch. I got confused with [this node](http://www.openstreetmap.org/node/1960994177), which is showing up in the dev tile and looked similar.